### DISCUSSION TITLE
呃，问题重复了，请忽略此提问，抱歉~

### DISCUSSION THREAD

**Discussion by hnwllrh** [the1812/Bilibili-Evolved#3776] [提问 / Q&A] [answered]
怎么无法下载1080+的FLV视频了？    dash格式的倒是能下载1080+，求指教~
![QQ截图20221024161524](https://user-images.githubusercontent.com/116548593/197480773-fcf76aa2-14db-4280-82ec-d91a3229c6f6.png)
![QQ截图20221024161540](https://user-images.githubusercontent.com/116548593/197480785-63545d6f-0dd5-4ae0-b160-dbbb88cd4776.png)

**Comment by Z-fly** [comment, marked as answer]
B站改版了
